test/tree/OperatorStateRetrieverUnit.tree

```diff
@@ -0,0 +1,35 @@
+.
+└── OperatorStateRetriever tree (*** denotes that integration tests are needed to validate path)
+    ├── when getOperatorState(IRegistryCoordinator,bytes32,uint32) is called
+    │   ├── when the given operator has never registered with the given registryCoordinator (their quorumBitmapHistory has length 0)
+    │   │   └── it should revert with "RegistryCoordinator.getQuorumBitmapIndicesAtBlockNumber: operator has no bitmap history"
+    │   └── when the given operator has registered for the first time with the given registryCoordinator
+    │       ├── after the given blockNumber
+    │       │   └── it should revert with "RegistryCoordinator.getQuorumBitmapIndicesAtBlockNumber: operator has no bitmap history at blockNumber"
+    │       └── before the given blockNumber
+    │           ├── and was not registered for any quorums at the given blockNumber
```

Comment:
I believe an invariant of the RegistryCoordinator is that a registered operator must have a nonzero bitmap.
So if the operator hasn't registered for any quorums i.e zero bitmap, they would be deregistered.